web/themes/joinup/prototype/sass/_mdl-hacks.sass

```diff
@@ -46,10 +46,26 @@
 .mdl-grid--center
   justify-content: center
 
-.join-collection-form
+.join-collection-form, .block-local-tasks-block
   .mdl-menu__item a
     white-space: nowrap
     color: $color-blue-default
   .mdl-menu__container
     margin-top: $field-margin
 
+.block-local-tasks-block
```

Comment:
This is styling related to mdl dropdown menu, so maybe it should stay in _mdl-hacks.sass